### ISSUE TITLE
feat: Update CSS to Semantic UI v2.3

### ISSUE THREAD

**Issue by super-rapidos** [edcarroll/ng2-semantic-ui#371]
# Update css version

Actually, https://edcarroll.github.io/ng2-semantic-ui/#/getting-started recommends to use semantic css 2.2.13, but this implies that some elements are missing compare to recent css versions (like icons).

## The update breaks some components

For example, when semantic 2.3.1 is used, modal positions are broken.

## Could you please consider to upgrade css version for ng2-semantic-ui ?

**Comment by miguelcaravantes** [comment]
i think the same, i had to update to 2.3.1 (https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.1/semantic.min.css) in my project when i see that it does showed some icons.

**Comment by yhnavein** [comment]
I fixed it in my PR #361 

I published it so you can use it normally by installing my version:
```
yarn add @yhnavein/ng2-semantic-us
```

Imports need to change, but after that it will work just fine
```
import { SuiModule } from '@yhnavein/ng2-semantic-ui';
```
